code/go/0chain.net/chaincore/chain/state.go

```diff
@@ -610,6 +624,11 @@ func (c *Chain) incrementNonce(sctx bcstate.StateContextI, fromClient datastore.
 		return err
 	}
 	logging.Logger.Debug("Updating nonce", zap.String("client", fromClient), zap.Int64("new_nonce", s.Nonce))
+
+	if err = c.emitEvent(sctx,stateToUser(fromClient, s), event.TagAddOrOverwriteUser); err != nil {
```

Comment:
Same as above

Reply:
@peterlimg it's ok now, we will differ this events in the next PR and they will have different types.